Generator/VariantGroupGenerator.php

```diff
@@ -0,0 +1,188 @@
+
+                return $this;
+            }
+        }
+
+        throw new Exception('There is no VARIANT group. \
+            Please add "group_types: ~" into your fixtures configuration file.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function generate(array $config, $outputDir, ProgressHelper $progress, array $options = null)
+    {
+        $this->faker = Faker\Factory::create();
+
+        $variantGroups = [];
+        $data          = [];
+
+        for ($i = 0; $i < $config['count']; $i++) {
+            $variantGroup = $this->generateVariantGroup($config);
+            $variantGroups[] = $variantGroup;
+            $data[] = $this->normalizeVariantGroup($variantGroup);
+
+            $progress->advance();
+        }
+
+        if (count($variantGroups) > 0) {
+            $this->writeCsvFIle($data, array_keys($data[0]), $outputDir);
+        }
+
+        return $variantGroups;
+    }
+
+    /**
+     * @param array $config
+     *
+     * @return GroupInterface
+     */
+    protected function generateVariantGroup($config)
+    {
+        $group = new Group();
+        $group->setType($this->variantGroupType);
+        $group->setCode($this->faker->word());
+
+        $attributeFaker = Faker\Factory::create();
+
+        $axisAttributes = [];
+        for ($i = 0; $i < $config['axis_count']; $i++) {
+            try {
+                $attribute = $attributeFaker->unique()->randomElement($this->optionsAttributes);
+            } catch (\OverflowException $e) {
+                throw new Exception(sprintf(
+                    'There is only %s attributes available for variant group axis, %s needed.',
+                    count($this->optionsAttributes),
+                    $config['axis_count']
+                ));
+            }
+            $axisAttributes[] = $attribute;
+        }
+
+        $group->setAxisAttributes($axisAttributes);
+
+        foreach ($this->locales as $locale) {
+            $translation = new GroupTranslation();
+            $translation->setLabel($this->faker->unique()->word());
+            $translation->setLocale($locale->getCode());
+            $group->addTranslation($translation);
+        }
+
+        return $group;
+    }
+
+    /**
+     * @param GroupInterface $variantGroup
+     *
+     * @return array
+     */
+    protected function normalizeVariantGroup(GroupInterface $variantGroup)
+    {
+        $axis = implode(',', array_map(function ($attribute) {
+            return $attribute->getCode();
+        }, $variantGroup->getAxisAttributes()->toArray()));
+
+        $result = [
+            'code' => $variantGroup->getCode(),
+            'axis' => $axis,
+            'type' => $variantGroup->getType()->getCode()
+        ];
+
+        /** @var GroupTranslation $translation */
+        foreach ($variantGroup->getTranslations() as $translation) {
+            $result[sprintf('label-%s', $translation->getLocale())] = $translation->getLabel();
+        }
+
```

Comment:
sounds good for axes, but we still not have variant group values here.
for instance,
```
code;label-en_US;type;axis;name-en_US;name-fr_FR
truc;Truc;VARIANT;size,color;my vg value 1;my vg value 3
```

To make it simple you can add only text attributes as value (because getting the data will be hard for any attribute type)